### ISSUE TITLE
Extremely High VRAM usable

### ISSUE THREAD

**Issue by TwistedTale** [34j/so-vits-svc-fork#237]
A clear and concise description of what the bug is:

- Extremely High VRAM usage

- All the time when the real time voice changer is running

Add any other context about the problem here:

- Everything works fine, I m just wondering if it's normal that it's using this much VRAM

![image](https://user-images.githubusercontent.com/76758164/229977055-af5eb0ef-4993-47f8-9444-9ddd098ab5a9.png)

**Comment by 34j** [comment]
Are you using the latest version?

**Comment by TwistedTale** [comment]
> 

yes I just updated to the latest, before it was pinned at max even when not actively converting voice

**Comment by TwistedTale** [comment]
![image](https://user-images.githubusercontent.com/76758164/230002191-9e8c4943-151a-4439-8833-9916877047af.png)

I also got this when running I dont know if it's related

**Comment by krzysiek546** [comment]
In my case latest versions can't take more than 3GB of VRAM (I have 6GB). I can't start second inference too because of lack of memory and it's impossible to stop realtime vc in gui. Sometimes after trying inference my screen flick couple times and system freezing. Older versions <~3.0.0 was using all VRAM and they didn't causing that problems.

![1](https://user-images.githubusercontent.com/74318268/232339655-c3dfea01-1b69-4ebd-920d-fa4c47a97636.PNG)
![alloc](https://user-images.githubusercontent.com/74318268/232339686-e47376ab-048f-475d-9c5f-ae88db60e852.PNG)

I'm using gtx 1660 super with latest nvidia drivers.
